docs/Developers/Writing_Documentation.rst

```diff
@@ -8,6 +8,11 @@ Documentation is written in reStructuredText and rendered by `Read the Docs
 <https://readthedocs.io>`_ to our lovely users.
 You can view the current user documentation on http://docs.coala.io.
 
+If you have never written a reStructuredText document before or
```

Comment:
The fact that the documentation is written in rst is mentioned before. I think "please go " formulation doesn't sound that friendly. Check the rephrase i did and tell me your opinion. :)